### DISCUSSION TITLE
Subtle flickering of brightness

### DISCUSSION THREAD

**Discussion by kacperMa25** [end-4/dots-hyprland#1008] [Q&A]
Hello, I keep experiencing flickering of brightness when screen is idle, I read in similar thread that disabling damage tracking helps with this, but that's not really my case, anyone knows what solution could be?

 Thanks for anyone that will reply.

**Comment by wylie13-home** [comment]
Does your monitor support freesync or some other variable refresh rate? I had the same issue on one monitor after the install. After days of searching, i realized it happened every time my monitor adjusted it's refresh rate. I noticed the issue never happened when YouTube videos were playing. The videos seemed to lock in a refresh rate on the monitor and it would hold steady.

**Comment by Satoxyan** [comment]
> Hello, I keep experiencing flickering of brightness when screen is idle, I read in similar thread that disabling damage tracking helps with this, but that's not really my case, anyone knows what solution could be?
> 
>  Thanks for anyone that will reply.

Edit your `~/.config/hypr/hyprland/general.conf`

Set the `vrr` to `0` or `false`, it would disabled the variable refresh rate
